src/Types/Any.purs

```diff
@@ -0,0 +1,18 @@
+module Types.Any
+  ( Any(..)
+  ) where
+
+import Prelude
+import Data.Generic.Rep (class Generic)
+import Data.Show.Generic (genericShow)
+
+-- Not sure if this is necessary but keep for now.
+-- | For converting a subset of polymorphic types to that which will work with
+-- | `PlutusData`.
+data Any
```

Comment:
Missing a kind signature

Reply:
There's no polymorphic variable here and I don't think we have standalone kinds, do we, if that's what you are referring to?

Reply:
Just implemented some changes if you could check.